src/models/category.js

```diff
@@ -0,0 +1,31 @@
+const { Schema, model } = require('mongoose')
+const { FIELD_CANNOT_BE_EMPTY } = require('~/consts/errors')
+const { CATEGORY } = require('~/consts/models')
+
+const categorySchema = new Schema(
+  {
+    name: {
+      type: String,
+      unique: true,
+      required: [true, FIELD_CANNOT_BE_EMPTY('name')]
+    },
+    appearance: {
+      icon: {
+        type: String,
+        required: [true, FIELD_CANNOT_BE_EMPTY('category icon')],
```

Comment:
You should to delete "required" field because you assigned default value. Mongoose will automatically assign a default value when the user does not pass an appropriate value when creating the document